app/_locales/en/messages.json

```
@@ -10,6 +10,43 @@
     "message": "$1 is not connected to any sites.",
     "description": "$1 is the account name"
   },
+  "connectedAccountsDescriptionSingular": {
+    "message": "You have 1 account connected to this site."
+  },
+  "connectedAccountsDescriptionPlural": {
+    "message": "You have $1 accounts connected to this site.",
+    "description": "$1 is the number of accounts"
+  },
+  "connectedAccountsEmptyDescription": {
+    "message": "MetaMask is not connected this site. To connect to a decentralized app (dapp), find the connect button on their site."
+  },
+  "primary": {
+    "message": "Primary"
+  },
+  "lastActive": {
+    "message": "Last active"
+  },
+  "connectThisAccount": {
+    "message": "Connect this account"
+  },
+  "switchToThisAccount": {
+    "message": "Switch to this account"
+  },
+  "disconnectThisAccount": {
+    "message": "Disconnect this account"
+  },
+  "viewConnectedSites": {
+    "message": "View more connected sites"
```

The copy on Figma is a little different:

```suggestion
    "message": "View other connected sites"
```

Saying "more" kinda implies the current site is connected too? It seems a little odd at least.

Maybe this is more clear:
```suggestion
    "message": "View connected sites"
```